### ISSUE TITLE
[BUG] Cloze Breaks When `}}` Encountered

### ISSUE THREAD

**Issue by LennyPhoenix** [st3v3nmw/obsidian-spaced-repetition#799]
When a cloze deletion (specifically using `==` as the delimiters) encounters the `}}` token, it seems to cut off the remaining card contents. This occurs even when the "Convert {{curly braces}} to clozes?" setting is disabled.

## To Reproduce

1. Ensure `Convert ==highlights== to clozes?` is enabled.
2. Create the following flashcard:
    ```md
    ==$\frac d {dx} (\arccos x)$== $=$ ==$-\frac 1 {\sqrt{1 - x^2}}$==
    ```
3. Review the flashcard, and note the broken LaTeX expression.

## Expected Behaviour

Card to render correctly in the review panel, like so:

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/57148878/9794e51e-461f-4cda-ae54-e66a93cf63ae)

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/57148878/32c2a4d5-3f97-4414-8937-6ab172b2a5c1)

## Screenshots

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/57148878/2d736d71-96e9-4897-8230-d9ddbb453242)

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/57148878/c3e8d2e4-fb2f-414d-81af-621dfb63ee96)

## Versions

-   OS: Bedrock (Arch) Linux
-   Obsidian version: v1.4.16
-   Plugin version: v1.10.5
-   Installer version: v1.4.16

## Additional Context

Workaround is to use `} }` in LaTeX expressions instead of `}}`.

**Comment by Stefanuk12** [comment]
This is just a guess, but I believe [this](https://github.com/st3v3nmw/obsidian-spaced-repetition/blob/master/src/QuestionType.ts#L116-L131) is causing the issue. All "terminators" are being replaced with nothing. I strongly recommend better use of your regex matches to fix this.

**Comment by ronzulu** [comment]
Hi  @LennyPhoenix, thanks for the detailed report, and @Stefanuk12 your guess was correct.

If you can test the following, and let me know if ok.
[main_0799A.zip](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/13500535/main_0799A.zip)


Cheers
Ronny

**Comment by ronzulu** [comment]
Hi @LennyPhoenix, wondering if you had a chance to try out the fix above.

**Comment by LennyPhoenix** [comment]
My apologies, it slipped my mind. I'll try it shortly.

**Comment by LennyPhoenix** [comment]
Sorry for the late response, but I believe this does fix the issue. Thanks!

**Comment by ronzulu** [comment]
Hi @LennyPhoenix the fix has been included in the new 1.11.0 release. Please close this issue at your convenience.

Cheers
Ronny

**Comment by LennyPhoenix** [comment]
Thanks !
